### PR TITLE
docs: add open api support to gomora dapp template

### PR DESCRIPTION
**Pull request by wowser20**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="OpenAPI UI" />
+    <title>Gomora-DApp API Documentation</title>
```
Comment (kabaluyot) [inline docs/index.html]: @wowser20 

should be => Gomora-dApp

File: docs/openapi.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
+{
+    "openapi": "3.0.2",
+    "info": {
+        "title": "Gomora DApp API",
```
Comment (kabaluyot) [inline docs/openapi.json]: Gomora-dApp

File: docs/openapi.json [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        "description": "A Gomora template for building EVM-compatible API and smart contract indexers and listeners",
+        "termsOfService": "https://github.com/Nuxify/gomora-dapp",
+        "contact": {
+            "name": "Gomora DApp API Developer",
```
Comment (kabaluyot) [inline docs/openapi.json]: Gomora-dApp

File: docs/openapi.json [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                    }
+                }
+            }
+        },
+        "/nft/replay": {
+            "get": {
+                "tags": [
+                    "nft"
+                ],
+                "summary": "Get Greeter Event",
```
Comment (kabaluyot) [inline docs/openapi.json]: Replay Greeter Contract Events

File: docs/openapi.json [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                "description": "Gets greeter event",
```
Comment (kabaluyot) [inline docs/openapi.json]: Replays greeter contract events by block number

**Review by kabaluyot — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 1 above)
Comment (wowser20) [reply docs/index.html]: working on it

**Review by kabaluyot — APPROVED** [review]

**Comment by kabaluyot** [comment]
LGTM

**Pull request merged**